docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '3'
+
+networks:
+  db-data-net:
+    driver: bridge
+
+services:
+  mysql:
+    image: mysql-local:2.0
+    build:
+      context: .
+      dockerfile: Dockerfile.mysql
+    container_name: mysql-container
+    ports:
+      - "3306:3306"
+    volumes:
+      - db-data:/var/lib/mysql
+    environment:
+      - MYSQL_ROOT_PASSWORD=1234
+      - MYSQL_DATABASE=app_db
+      - MYSQL_USER=app_user
+      - MYSQL_PASSWORD=1234
+    networks:
+      - db-data-net
+
+  pythonapp:
+    image: todo-app:2.0
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: todoapp-container
+    ports:
+      - "8080:8080"
+    environment:
+      - PYTHONUNBUFFERED=1
+    networks:
+      - db-data-net
+    depends_on:
+      - mysql
+    restart: unless-stopped
+volumes:
+  db-data:
```

Comment:
Add EoF(End-of-File), need to add empty line at the end
![image](https://github.com/mate-academy/devops_todolist_docker_core_task_3_docker_compose/assets/87720583/15f13e7c-a632-4f01-b1bf-3283e4d09783)